app/src/org/commcare/android/models/AndroidSessionWrapper.java

```diff
@@ -131,72 +125,6 @@ public void setFormRecordId(int formRecordId) {
         this.formRecordId = formRecordId;
     }
     
-    /**
-     * Registers the instance data returned from form entry about this session, and specifies
-     * whether the returned data is complete 
-     *
-     * @param uri points to the instance we want to register with the session
-     * @param c A cursor which points to at least one record of an ODK instance.
-     * @return True if the record in question was marked completed, false otherwise
-     * @throws IllegalArgumentException If the cursor provided doesn't point to any records,
-     * or doesn't point to the appropriate columns
-     */
-    public boolean beginRecordTransaction(Uri uri, Cursor c) throws IllegalArgumentException {
```

Comment:
No longer needed since registering the record with the session can all be done in one place.